drivers/quota/projectquota_supported.go

```diff
@@ -370,6 +371,20 @@ func setProjectID(targetPath string, projectID uint32) error {
 		return fmt.Errorf("failed to set projid for %s: %w", targetPath, errno)
 	}
 
+	// While we are setting the PROJINHERIT flag, it only applies to files
+	// and directories created after the fact.
```

Comment:
Nothing here changes properties of non-directories. Doesn’t the project ID have to be set for every file as well?

Reply:
No, it is a per-directory construct

Reply:
I agree the _flag_ is per-directory. The project ID itself?